packages/vscode-dependencies-validation/test/commands.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { dirname } from "path";
+import * as proxyquire from "proxyquire";
+import { createSandbox, SinonMock, SinonSandbox } from "sinon";
+import { outputChannel, diagnosticCollection } from "./vscodeMocks";
+import { NpmCommandConfig } from "@sap-devx/npm-dependencies-validation";
+import { fixAllDepIssuesCommand } from "../src/commands";
+
+const npmDepsValidationProxy = {
+  invokeNPMCommand() {
+    return Promise.reject("invokeNPMCommand method is not implemented");
+  },
+  "@noCallThru": true,
+};
+
+const diagnosticsProxy = {
+  refreshDiagnostics() {
+    return Promise.reject("refreshDiagnostics method is not implemented");
+  },
+  "@noCallThru": true,
+};
+
+describe("commands unit test", () => {
+  let sandbox: SinonSandbox;
+  let outputChannelMock: SinonMock;
+  let npmDepsValidationMock: SinonMock;
+  let diagnosticsMock: SinonMock;
+
+  before(() => {
```

Comment:
what does it mean that the sandbox is shared between multiple tests?
could it cause shared state between tests?

Reply:
it is ok, see here: https://sinonjs.org/releases/latest/sandbox/